addons/website_slides/views/slide_channel_views.xml

```diff
@@ -111,6 +111,18 @@
                                     <field name="allow_comment"/>
                                 </group>
                             </page>
+                            <page string="Karma Rules">
```

Comment:
name="karma_rules"

Reply:
Yes please. Having `name` on the main objects makes it so much easier to inherit and add clean code.